tools/prowgen/pkg/testdata/.base.yaml

```diff
@@ -105,3 +105,8 @@ requirement_presets:
     - "common"
     - "args"
     - "that are reusable"
+  secrets:
+    secrets:
```

Comment:
two levels?

Reply:
The first one is the requirement name, in non-test it will be something like `release:`. But probably could have better naming